src/main/java/apoc/result/VirtualNode.java

```diff
@@ -136,6 +136,12 @@ public VirtualRelationship createRelationshipTo(Node node, RelationshipType rela
         return rel;
     }
 
+    public VirtualRelationship createRelationshipFrom(Node start, RelationshipType relationshipType) {
```

Comment:
Why? is this really needed? it's not part of the Node interface.

Reply:
Unfortunately, we can't create a relationship from a "real" node to a virtual

Reply:
The only way is to create it a reverse way